lib/compat/wordpress-6.4/script-loader.php

```diff
@@ -158,9 +158,13 @@ function _gutenberg_get_iframed_editor_assets_6_4() {
 		}
 	}
 
+	// Remove the deprecated `print_emoji_styles` handler.
+	// It avoids breaking style generation with a deprecation message.
+	remove_action( 'wp_print_styles', 'print_emoji_styles' );
```

Comment:
While this works, it feels a little hacky to me. Why do we need to remove core actions in order for the output to work correctly?

Reply:
The deprecation `print_emoji_styles` function is still hooked into `wp_print_styles` for backward compatibility. Core handles its removal differently, so we must resort to manual removal.

I'm open to suggestions for a better fix.